### PR TITLE
fedora_bot: support cockpit-composer

### PR DESCRIPTION
**Pull request by jkozol**

### CODE REVIEW

File: fedora_bot.py [hunk 1]
```diff
@@ -125,7 +125,8 @@ def schedule_fedora_builds(args,component,fedoras,missing_updates):
 
     kojis = { "osbuild": "https://koji.fedoraproject.org/koji/packageinfo?packageID=29756",
               "osbuild-composer": "https://koji.fedoraproject.org/koji/packageinfo?packageID=31032",
-              "koji-osbuild": "https://koji.fedoraproject.org/koji/packageinfo?packageID=32748" }
+              "koji-osbuild": "https://koji.fedoraproject.org/koji/packageinfo?packageID=32748",
+              "cockpit-composer":  "https://koji.fedoraproject.org/koji/packageinfo?packageID=28261" }
```
Comment (ochosi) [inline fedora_bot.py:129]:
```suggestion
              "cockpit-composer": "https://koji.fedoraproject.org/koji/packageinfo?packageID=28261" }
```

File: fedora_bot.py [hunk 2]
```diff
@@ -248,7 +249,7 @@ def get_fedora_releases():
 
 def main():
     """Main function"""
-    components = ['osbuild','osbuild-composer','koji-osbuild']
+    components = ['osbuild','osbuild-composer','koji-osbuild', 'cockpit-composer']
```
Comment (ochosi) [inline fedora_bot.py:252]:
```suggestion
    components = ['osbuild','osbuild-composer','koji-osbuild','cockpit-composer']
```

**Comment by ochosi** [comment]
Looks good - thanks!
I guess as soon as you're ready in cockpit-composer to switch to our automation (i.e. set up packit, drop the Makefile and let the specfile contain all the generated changes like the release version) we can go ahead merge this.

That said, fedora-bot is non-destructive, all it does is scheduling builds for latest Fedora dist-git versions so in theory it could be merged anytime.

**Review by ochosi — CHANGES_REQUESTED** [review]

**Comment by jkozol** [comment]
Will fix these whitespace issues. 

Also, I am going to temporarily leave this as a draft until I talk more with cockpit. I discussed this briefly with @kkoukiou and she recommended that we continue to use cockpit's release tools just as their other plugins do.

**Pull request closed without merging**